authors/apps/authentication/views.py

```diff
@@ -41,17 +41,15 @@ def post(self, request):
 
         email = serializer.data['email']
         username = serializer.data['username']
-        token = serializer.data['token']
```

Comment:
add this token as a param in the url. That is, the final url should be
`
https://ah-fulldeck-frontend-staging.herokuapp.com?token=some-long-token
`